src/pages/LoginPage.tsx

```diff
@@ -2,8 +2,15 @@ import React, { Component } from 'react';
 
 class LoginPage extends Component {
     render() {
-      return <div>Log in stuff will go here</div>
+        // TODO: Redirect home after submission
```

Comment:
This is for my own edification, but how does the  form know to post to `/login`?

Reply:
o.O no idea!